src/package/nuspec/Microsoft.NET.Test.Sdk.nuspec

```diff
@@ -19,14 +19,6 @@
                 branch="$BranchName$"
                 commit="$CommitId$" />
     <dependencies>
-      <group targetFramework="uap10.0">
```

Comment:
Shall we add `netstandard2.0` in replacement of `uap10.0`?

Reply:
don't we have netstandard2.0 in the layout?

Reply:
I have just checked and we have only `net462`, `netcoreapp3.1` and before `uap10.0`. That's why I think adding `netstandard2.0` would be useful.

Reply:
@nohwnd Any guidance here?